### PR TITLE
Fr/feature/animations

### PR DESCRIPTION
**Pull request by RavenInd**
- Desktop animation
- Mobile animation
- Theme switcher styles

### CODE REVIEW

File: docs/src/components/DocsNavigation.tsx [hunk 1]
```diff
@@ -172,9 +197,17 @@ const DocsNavigation: FC<Props> = ({ isSidebarOpen, onSidebarOpen }) => {
           socialsItems={socialsItems}
         />
         <ThemeButton onClick={changeTheme} aria-label="change color theme">
-          {theme === ThemesEnum.AUTO && "Auto "}
-          <ThemeIconWrapper data-theme={theme}>
-            <ThemeSwitcher />
+          {theme === ThemesEnum.AUTO && "A "}
```
Comment (usulpro) [inline docs/src/components/DocsNavigation.tsx]: Can we plese wrap it into span and remove space (setup margin instead)

File: docs/src/components/landingBlocks/HeroBlock.tsx [hunk 2]
```diff
@@ -84,42 +87,12 @@ const HeroBlock = ({ content }: Props) => {
...
-                height: "100%",
-                width: "100%",
-                textAlign: "center",
-              }}
-            >
-              MOBILE ANIMATION MOCK
-            </div>
-          )}
-        </Block>
+        <Block>{isDesktop ? <DesktopAnimation /> : <MobileAnimation />}</Block>
```
Comment (usulpro) [inline docs/src/components/landingBlocks/HeroBlock.tsx]: This affects LCP metric

Comment (usulpro) [reply docs/src/components/landingBlocks/HeroBlock.tsx]: image

**Review by usulpro — CHANGES_REQUESTED** [review]

File: docs/src/components/MobileAnimation.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    ? 0
+    : anchorPosition - defaultAnchorPosition;
+};
+
+const MobileAnimation = () => {
+  const [anchorPosition, setAnchorPosition] = useState(0);
+  const anchorRef = useRef<HTMLElement>(null);
+
+  useEffect(() => {
+    const main = document.getElementById("main");
```
Comment (usulpro) [inline docs/src/components/MobileAnimation.tsx]: can't we get using React useRef API?

**Review by usulpro — CHANGES_REQUESTED** [review]

File: docs/src/components/landingBlocks/HeroBlock.tsx (hunk 2 above)
Comment (RavenInd) [reply docs/src/components/landingBlocks/HeroBlock.tsx]: Used css instead

File: docs/src/components/MobileAnimation.tsx (hunk 3 above)
Comment (RavenInd) [reply docs/src/components/MobileAnimation.tsx]: Switched to React useRef API

File: docs/src/components/DocsNavigation.tsx (hunk 1 above)
Comment (RavenInd) [reply docs/src/components/DocsNavigation.tsx]: I just deleted this space.
I believe that we don't need more space here

**Review by usulpro — APPROVED** [review]

**Pull request merged**